package.json

```diff
@@ -130,12 +130,7 @@
           "configFile": true
         }
       ],
-      [
```

Comment:
The config validation is stricter now and the configFile option doesn't exist for typescript.